src/config/config.js

```diff
@@ -20,6 +21,7 @@ const getDatabaseConfig = () => {
         password: process.env.PROD_DATABASE_PASSWORD ?? "",
         host: process.env.PROD_DATABASE_HOST ?? "",
         port: process.env.PROD_DATABASE_PORT ?? "",
+        dialect: "postgres",
```

Comment:
same as here

Reply:
This line also, I think we may supply the dialect  run `npm run migrate` or `npm run seed`,

@fab-ryan, can we get another alternative